### ISSUE TITLE
request to add 'kissmanga'

### ISSUE THREAD

**Issue by codeduph** [NguyenDanPhuong/MangaRipper#28]
is it possible to add http://kissmanga.com/. I really prefer this site for a long time and its really famous site for reading manga... (also anime, drama, cartoon add to their name)

**Comment by amayra** [comment]
thanks to cloudflare browser verification i don't think so 
ps: i don't think this will happen soon but i love KissManga so im trying to do something about it

**Comment by codeduph** [comment]
it was supported in FMD and DomDomSoft? So, I don't think couldflare is a problem.

**Comment by amayra** [comment]
nope try this app now and to fix this we need KissManga's cookies to bypass couldflare 
KissManga ni my top priority list when i have free time
KissManga owner : Deal With It

**Comment by hoanghuy89** [comment]
Thanks. I tried image quality comparison before and kissmanga has better quality than other site like mangafox. Mangahere is double of mangafox, nothing more.

**Comment by amayra** [comment]
@hoanghuy89 kissmanga has better quality but bad Translators 
There you go. Those are probably the best online sites (bato.to,mangatown.com,mangastream followed by kissmanga), and I'd avoid Mangafox, Mangahere,mangapanda....etc.

**Comment by JedBurke** [comment]
I experimented with adding KissManga support since I came across #12. Encountered the CloudFlare bit myself and thought I was going to have to write something to parse the 'obfuscated' JS. Luckily enough, someone else already [did](https://github.com/elcattivo/CloudFlareUtilities).

Not debating the quality of the site or whether *should* be added, the point is that it's possible. I did my [work](https://github.com/JedBurke/MangaRipper/tree/master) on a modified version of the 2011 build. Haven't seen the updated source as of yet, so it might be easier there than here; as a few things had to be adjusted to get it to work.
